enterprise/course_catalog_api.py

```diff
@@ -64,6 +64,30 @@ def get_all_catalogs(self):
         """
         return self._load_data('catalogs', default=[])
 
+    def get_single_catalog_courses(self, catalog_id):
```

Comment:
why `get_single_catalog_courses` instead of `get_catalog_courses`?  Is the latter already in-use?